docs/backwardcompatibilityml/topics/using_the_tensorflow_interface.rst

```diff
@@ -0,0 +1,93 @@
+.. _using_the_tensorflow_interface:
+
+The Tensorflow Interface
+=================================================================
+
+There are two ways in which one may use the notion of the Backward
+Compatibility loss with respect to the Tensorflow framework.
+
+- Creating and training a model subclassed from BCNewErrorCompatibilityModel
+- Training a general model h2 using Backward Compatibility Loss
+
+The first method for achieving this allows one to leverage the existing
+``h2.fit(...)`` method in order to train your model. However it requires
+that ``h2`` was instantiated from a model subclassed from BCNewErrorCompatibilityModel.
+If you already had a model instantiated from a distinct model class, then
+you mayneed to go through soe effort to extract the layers you need
```

Comment:
Space between "may" and "need."  Change "through soe" to "to some"

Reply:
Done.